prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/KS/KS-receiver-transform.yml

```diff
@@ -0,0 +1,59 @@
+hl7Class: ca.uhn.hl7v2.model.v251.message.ORU_R01
+
+extends: classpath:/metadata/hl7_mapping/ORU_R01/ORU_R01-base.yml
+
+elements:
+
+  - name: xtn-use-code-from-system
+    condition: '%resource.system = "email"'
+    value: [ '"NET"' ]
+    hl7Spec: [ '%{hl7TelecomPath}-2' ]
+
+  - name: ks-message-encoding-characters
+    condition: true
+    value: [ '"^~\\&"' ]
+    hl7Spec: [ 'MSH-2' ]
+
+  - name: ks-name-of-the-receiving-application
+    condition: true
+    value: [ '"KSDOH"' ]
+    hl7Spec: [ 'MSH-5-1' ]
+
+  - name: ks-OID-of-the-receiving-application
+    condition: true
+    value: [ '"2.16.840.1.114222.4.3.3.36"' ]
+    hl7Spec: [ 'MSH-5-2' ]
+
+  - name: ks-indicates-that-5.2-is-an-OID
+    condition: true
+    value: [ '"ISO"' ]
+    hl7Spec: [ 'MSH-5-3' ]
+
+  - name: ks-receiving-facility-namespace-ID
+    condition: true
+    value: [ '"KDHE"' ]
+    hl7Spec: [ 'MSH-6-1' ]
+
+  - name: ks-receiving-facility-universal-ID
+    condition: true
+    value: [ '"2.16.840.1.114222.4.3.2.2.1.163"' ]
+    hl7Spec: [ 'MSH-6-2' ]
+
+  - name: ks-receiving-facility-universal-ID-Type
+    condition: true
+    value: [ '"ISO"' ]
+    hl7Spec: [ 'MSH-6-3' ]
+
+  - name: ks-patient-race-coding-system
+    resource: 'Bundle.entry.resource.ofType(Patient).extension("http://ibm.com/fhir/cdm/StructureDefinition/local-race-cd").value.coding'
+    condition: '%resource.code.exists()'
+    value: [ '%resource.system.getCodingSystemMapping()' ]
+    hl7Spec: [ '/PATIENT_RESULT/PATIENT/PID-10-3' ]
+
+  - name: observation-result-with-aoe
+    # Grab only the AOE observations from ServiceRequest.supportingInfo NOT associated with a specimen
```

Comment:
We are not suppressing AOEs in the covid pipeline. Why are we suppressing them in the UP?

Reply:
KS doesn't need AOE.